### PR TITLE
Track EA APIs

### PR DESCRIPTION
**Pull request by 333fred**
Updates to the latest prerelease of our analyzers, which include the ability to track `internal` APIs. We will now track the APIs we ship via our EA packages, much like we do with our regular Public APIs. This will hopefully alert us to accidental refactorings that will break an insertion, or remind us to ensure that changes that need a dual insertion are known about ahead of time.

### CODE REVIEW

**Comment by 333fred** [comment]
@sharwell @dotnet/roslyn-infrastructure for reviews please.

**Review by genlu — APPROVED** [review]
Nice!

**Review by mavasani — APPROVED** [review]

File: src/Tools/ExternalAccess/FSharp/Navigation/IFSharpDocumentNavigationService.cs [hunk 1]
```diff
@@ -20,13 +20,15 @@ internal interface IFSharpDocumentNavigationService : IWorkspaceService
         [Obsolete("Call overload that takes a CancellationToken", error: false)]
         bool CanNavigateToLineAndOffset(Workspace workspace, DocumentId documentId, int lineNumber, int offset);
         [Obsolete("Call overload that takes a CancellationToken", error: false)]
+#pragma warning disable RS0060 // API with optional parameter(s) should have the most parameters amongst its public overloads
```
Comment (sharwell) [inline src/Tools/ExternalAccess/FSharp/Navigation/IFSharpDocumentNavigationService.cs:23]: 💡 These two lines should be reversed if the `#pragma warning` is used for a group  of methods

**Review by sharwell — APPROVED** [review]

File: src/Tools/ExternalAccess/FSharp/Navigation/IFSharpDocumentNavigationService.cs (hunk 1 above)
Comment (333fred) [inline src/Tools/ExternalAccess/FSharp/Navigation/IFSharpDocumentNavigationService.cs:23]:
```suggestion
#pragma warning disable RS0060 // API with optional parameter(s) should have the most parameters amongst its public overloads
        [Obsolete("Call overload that takes a CancellationToken", error: false)]
```

**Pull request merged**